NEWS.md

```diff
@@ -1,3 +1,9 @@
+# inbodb 0.0.6
+
+* new functions `get_taxonlijsten_lists()`, `get_taxonlijsten_features()` and
+  `get_taxonlijsten_items()` to query the 'D0156_00_Taxonlijsten' database
```

Comment:
I would suggest to also add your vignet, as I saw a nice example [here](https://inbo.github.io/checklist/news/index.html).  But I encounter problems installing the vignettes here, so I can't test yet if it works to refer to it as `vignette("get_data_taxonlijsten", package = "inbodb")`.  Anyway, just imagine it would work like that, I'll try to unrafel the technical problems after this review.

Reply:
Ok, I installed it locally with vignettes, and the suggested code indeed leads to the vignette.  So it should just work fine when refering to this.
Just for the information: installing your most recent version from github can be done by:
`devtools::install_github("inbo/inbodb@functions_taxonlijsten", build_vignettes = TRUE)`